We need one patch on top of 0.3.8 for compatibility with Werkzeug 1.0.

Taken from upstream:
https://github.com/flask-restful/flask-restful/commit/73376a488907af3042b52678ac4c23f8a8911e5b

diff --git a/tests/test_api.py b/tests/test_api.py
index f7f8e661..6795d362 100644
--- a/tests/test_api.py
+++ b/tests/test_api.py
@@ -445,7 +445,9 @@ def test_handle_non_api_error(self):
 
         resp = app.get("/foo")
         self.assertEquals(resp.status_code, 404)
-        self.assertEquals('text/html', resp.headers['Content-Type'])
+        # in newer versions of werkzeug this is `text/html; charset=utf8`
+        content_type, _, _ = resp.headers['Content-Type'].partition(';')
+        self.assertEquals('text/html', content_type)
 
     def test_non_api_error_404_catchall(self):
         app = Flask(__name__)
diff --git a/tests/test_reqparse.py b/tests/test_reqparse.py
index 2f1fbedf..9776f17c 100644
--- a/tests/test_reqparse.py
+++ b/tests/test_reqparse.py
@@ -2,9 +2,9 @@
 import unittest
 from mock import Mock, patch
 from flask import Flask
-from werkzeug import exceptions, MultiDict
+from werkzeug import exceptions
 from werkzeug.wrappers import Request
-from werkzeug.datastructures import FileStorage
+from werkzeug.datastructures import FileStorage, MultiDict
 from flask_restful.reqparse import Argument, RequestParser, Namespace
 import six
 import decimal
